gas-tank/balance-services.md

```diff
@@ -0,0 +1,540 @@
+    tokenAddress: "string", // [!code highlight]
+    chain: "string", // [!code highlight]
+    owner: "string", // [!code highlight]
+    spender: "string", // [!code highlight]
+    value: "string", // [!code highlight]
+    deadline: 0, // [!code highlight]
+    v: 0, // [!code highlight]
+    r: "string", // [!code highlight]
+    s: "string" // [!code highlight]
+  },
+})
+  .then((response) => {
+    // Balance increased successfully
+  })
+  .catch((error) => {
+    // Catch & handle the error
+  });
+```
+
+:::
+
+### Withdraw balance
+
+This endpoint allows users to withdraw their balance from the Gas Tank platform.
+
+To withdraw the balance, you need to sign the message with the private key of the address you want to withdraw the balance from. The `signature` field is the signature of the message signed by the address owner.
+
+Same as the [Increase balance](#increase-balance), you can sign a string of data using [ethers.js](https://docs.ethers.org/v5/getting-started/), below is an example of [how to sign a message using ethers.js](https://docs.ethers.org/v5/getting-started/#getting-started--signing) and withdraw the balance.
+
+::: code-group
+
+```javascript [Request]
+import { ethers } from "ethers";
+
+const GAS_TANK_ENDPOINT = "https://gas-tank.xdefi.services";
+const message = "0x1234"; // Message to sign // [!code highlight]
+const privateKey = "0x1234"; // Private key of the address // [!code highlight]
+const wallet = new ethers.Wallet(privateKey);
+const signature = await wallet.signMessage(message);
+
+await fetch(`${GAS_TANK_ENDPOINT}/balances/withdraw`, {
+  method: "POST",
+  headers: {
+    "Content-Type": "application/json"
+    "Authorization": `Bearer ${jwtToken}`, // JWT token // [!code highlight]
+  },
+  body: {
+    address: "string", // [!code highlight]
+    tokenAddress: "string", // [!code highlight]
+    amount: "string", // [!code highlight]
+    chain: "string", // [!code highlight]
+    recipient: "string", // [!code highlight]
+    message: "string", // [!code highlight]
+    signature: "string" // [!code highlight]
+  },
+})
+  .then((response) => {
+    // Balance withdrawn successfully
+  })
+  .catch((error) => {
+    // Catch & handle the error
+  });
+```
+
+:::
+
+### Internal transfer
+
+This endpoint allows users to create an internal transfer task on the Gas Tank platform.
+
+::: code-group
+
+```javascript [Request]
+const GAS_TANK_ENDPOINT = "https://gas-tank.xdefi.services";
+
+await fetch(`${GAS_TANK_ENDPOINT}/balances/transfer`, {
+  method: "POST",
+  headers: {
+    "Content-Type": "application/json",
+    Authorization: `Bearer ${jwtToken}`, // JWT token // [!code highlight]
+  },
+  body: {
+    address: "string", // [!code highlight]
+    tokenAddress: "string", // [!code highlight]
+    amount: "string", // [!code highlight]
+    chain: "string", // [!code highlight]
+    recipient: "string", // [!code highlight]
+    message: "string", // [!code highlight]
```

Comment:
Can you specify what is the messages to sign here (how to get it? via API) and how to sign it